lambdas/matching/src/matching-handler.ts

```diff
@@ -40,10 +40,21 @@ export class MatchingHandler implements LambdaInterface {
       const txn = response.headers.get("x-amz-cf-id") ?? "";
       addLogEntry(event, txn, context);
       const contentType = response.headers.get("content-type");
+
       if (contentType?.includes("application/json")) {
+        let responseJson;
```

Comment:
```suggestion
        let response;
```
This might not always be JSON

Reply:
Made it `responseBody`